src/Classes/Item.lua

```diff
@@ -283,28 +285,37 @@ function ItemClass:ParseRaw(raw)
 					importedLevelReq = tonumber(specVal)
 				elseif specName == "LevelReq" then
 					self.requirements.level = tonumber(specVal)
-				elseif specName == "Has Alt Variant" then
-					self.hasAltVariant = true
-				elseif specName == "Has Alt Variant Two" then
-					self.hasAltVariant2 = true
-				elseif specName == "Has Alt Variant Three" then
-					self.hasAltVariant3 = true
-				elseif specName == "Has Alt Variant Four" then
-					self.hasAltVariant4 = true
-				elseif specName == "Has Alt Variant Five" then
-					self.hasAltVariant5 = true
+				elseif specName:match("Has Alt Variant") then
+					local variantName = specName == "Has Alt Variant" and "One" or specName:gsub("Has Alt Variant ","")
+					self.variantList = self.variantList or { Base = { } }
+					self.variantList[variantName] = {}
+					if specVal:match("%(%d+,%d+%)") then
```

Comment:
```suggestion
					if specVal:match("%(%d+%-%d+%)") then
```
Ranges typically use (x-y) notations might be a better format?

Reply:
this was incase I wanted to mess with ranges to make it "min, max, step", but ended up not needing it at the time so can change it to use "-" if I dont need step